Template/LO/SubProcesses/myamp.f

```diff
@@ -285,6 +286,7 @@ subroutine set_peaks
       double precision stot,m1,m2
       common/to_stot/stot,m1,m2
 
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc